tools/scoring/score/dimensions/unit_mapping.py

```diff
@@ -14,14 +14,17 @@
 """Core component."""
 
 from score.dimensions.dimension import Dimension
-from score.constants import FileTypes
+from score.constants import FileTypes, DimensionCategories
 
 PROPOSED, SOLUTION = FileTypes
 
 
 class UnitMapping(Dimension):
   """Quantifies how accurately the proposed file
   mapped dimensional units for relevant fields."""
+
+  category = DimensionCategories.SIMPLE
```

Comment:
can you please update https://github.com/google/digitalbuildings/blob/master/tools/scoring/score/constants.py#L26
to clarify which ones are simple and which are complex?

It seems you differentiate them based on how you are calculating them?

If it is simple ---> simple processing ?
complex --> the aglorithm needs to do some extra steps to identify things?

Reply:
You got it. I'll work on expanding and unifying the explanation.